intellij/src/saros/intellij/ui/views/buttons/FollowButton.java

```diff
@@ -141,6 +149,9 @@ private JMenuItem createItemForUser(User user) {
 
     JMenuItem menuItem = new JMenuItem(menuItemPrefix + userNameShort);
```

Comment:
In the other class you use a JBMenuItem. Any special reason to use a JMenuItem here ?

Reply:
I just forgot. Fixed.